.github/workflows/ci.yml

```
@@ -360,14 +363,89 @@ jobs:
+    - name: Build Docker image
+      run: |
+        docker build --help
+        docker build . --file ${{ env.DOCKERFILE_PHP_BETA }} --tag "${{ env.IMAGE_PHP_BETA }}-${{ env.DATE }}"
+    - name: Start Docker container
+      run: |
+        docker images
+        docker run --help
+        docker run -d --name php_container "${{ env.IMAGE_PHP_BETA }}-${{ env.DATE }}"
+    - name: Verify build
+      run: docker exec php_container cat /etc/os-release
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)